runtime/near-evm-runner/src/types.rs

```diff
@@ -96,6 +102,124 @@ pub struct MetaCallArgs {
     pub input: Vec<u8>,
 }
 
+#[derive(Debug, Eq, PartialEq)]
+pub struct EthTransaction {
+    pub nonce: U256,
+    pub gas_price: U256,
+    pub gas: U256,
+    pub to: Option<Address>,
+    pub value: U256,
+    pub data: Vec<u8>,
+}
```

Comment:
Can you please add comments to the fields

Reply:
Resolved in 311b3c8ce1c18d1662e5549575f2b3ad8657a21a.